src/main/kotlin/com/workos/users/UsersApi.kt

```diff
@@ -463,4 +462,110 @@ class UsersApi(private val workos: WorkOS) {
 
     return workos.post("/users/password_reset_challenge", CreatePasswordResetChallengeResponse::class.java, config)
   }
+
+  /**
+   * Parameters for the [verifyEmail] method.
+   */
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  class VerifyEmailOptions @JvmOverloads constructor(
+    @JsonProperty("magic_auth_challenge_id") val magicAuthChallengeId: String,
+    @JsonProperty("code") val code: String
+  ) {
+    init {
+      require(magicAuthChallengeId.isNotBlank()) { "Magic Auth Challenge ID is required" }
+      require(code.isNotBlank()) { "Code is required" }
+    }
+
+    /**
+     * Builder class for [VerifyEmailOptions].
+     */
+    class VerifyEmailOptionsBuilder {
+      private lateinit var magicAuthChallengeId: String
+      private lateinit var code: String
+
+      /**
+       * Sets the magic auth challenge id.
+       */
+      fun magicAuthChallengeId(value: String) = apply { this.magicAuthChallengeId = value }
+
+      /**
+       * Sets the code.
+       */
+      fun code(value: String) = apply { this.code = value }
+
+      /**
+       * Creates a [VerifyEmailOptions] with the given builder parameters.
+       */
+      fun build(): VerifyEmailOptions {
+        return VerifyEmailOptions(magicAuthChallengeId, code)
+      }
+    }
+
+    /**
+     * @suppress
+     */
+    companion object {
+      @JvmStatic
+      fun builder(): VerifyEmailOptionsBuilder {
+        return VerifyEmailOptionsBuilder()
+      }
+    }
+  }
+
+  /**
+   * Verifies an email.
+   */
+  fun verifyEmail(verifyEmailOptions: VerifyEmailOptions): User {
+    val config = RequestConfig.builder().data(verifyEmailOptions).build()
+    return workos.post("/users/verify_email", User::class.java, config)
+  }
+
+  /**
+   * Parameters for the [sendVerificationEmail] method.
+   */
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  class SendVerificationEmailOptions @JvmOverloads constructor(
+    @JsonProperty("user_id") val userId: String
+  ) {
+    init {
+      require(userId.isNotBlank()) { "User ID is required" }
+    }
+
+    /**
+     * Builder class for [SendVerificationEmailOptions].
+     */
+    class SendVerificationEmailOptionsBuilder {
+      private lateinit var userId: String
+
+      /**
+       * Sets the user ID.
+       */
+      fun userId(value: String) = apply { this.userId = value }
+
+      /**
+       * Creates a [SendVerificationEmailOptions] with the given builder parameters.
+       */
+      fun build(): SendVerificationEmailOptions {
+        return SendVerificationEmailOptions(userId)
+      }
+    }
+
+    /**
+     * @suppress
+     */
+    companion object {
+      @JvmStatic
+      fun builder(): SendVerificationEmailOptionsBuilder {
+        return SendVerificationEmailOptionsBuilder()
+      }
+    }
+  }
+
+  /**
+   * Sends a verification email to a user.
+   */
+  fun sendVerificationEmail(options: SendVerificationEmailOptions): User {
```

Comment:
The response type for this endpoint has been updated. It no longer returns a user, rather it returns the same payload as the `users/magic_auth/send` endpoint. https://workos-test.com/docs/reference/user-management/email-verification/send_verification_email